### PR TITLE
NXP-32004: Document scheduler endpoint

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

**Comment by guirenard** [comment]
see https://github.com/nuxeo/nuxeo-lts/pull/1411

File: src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Stop the scheduler
+
+```
+PUT /management/stop
+```
+
+### Response
+
+Always 204, this stops the scheduler service cluster wild. The stops survives a server restart.
```
Comment (bdelbosc) [inline src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md]: wide :)

File: src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Status Codes
+
+- 204 _NO_CONTENT_
+
+### Sample
+
+```curl
+curl -XPUT -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/managementscheduler/stop
```
Comment (bdelbosc) [inline src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md]: management/scheduler

File: src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Start the scheduler
+
+```
+PUT /management/start
+```
+
+### Response
+
+Always 204, this starts the scheduler service cluster wild.
```
Comment (manonlumeau) [inline src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md]:
```suggestion
Always 204, this starts the scheduler service cluster-wide.
```

File: src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+## Stop the scheduler
+
+```
+PUT /management/stop
+```
+
+### Response
+
+Always 204, this stops the scheduler service cluster-wide. The stops survives a server restart.
```
Comment (manonlumeau) [inline src/rest-api/endpoints/management-endpoint/scheduler-endpoint.md]:
```suggestion
Always 204, this stops the scheduler service cluster-wide. The stop survives a server restart.
```

**Pull request merged**